python/nav/mibs/itw_mibv4.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2008-2011 Uninett AS
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 2 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+A class that tries to retrieve all sensors from WeatherGoose II.
+
+Uses the vendor-specifica IT-WATCHDOGS-V4-MIB to detect and collect
+sensor-information.
+
+Please note:
+This is NOT a full implementaion of the IT-WATCHDOGS-V4-MIB. Only the internal
+sensors of the box are implemented. The box can be extended with additional
+external sensors, but these are not implemented because we did not have any
+external sensors available at the time of this implementation.
+"""
+from django.utils.six import itervalues
+from twisted.internet import defer
+
+from nav.mibs import reduce_index
+from nav.mibs import mibretriever
+from nav.models.manage import Sensor
+from nav.oids import OID
+
+from .itw_mib import for_table
+
+
+class ItWatchDogsMibV4(mibretriever.MibRetriever):
+    """A class that tries to retrieve all sensors from WeatherGoose II"""
```

Comment:
This is simply not true, as stated elsewhere in the same module. Please update the docstring.